### PR TITLE
DOC: Add module synopsis

### PR DESCRIPTION
**Pull request by bashtage**
Make changes that allow the py-modindex to correctly build

[skip ci]

### CODE REVIEW

File: docs/source/gee.rst [hunk 1]
```diff
@@ -84,6 +85,8 @@ Dependence Structures
 The dependence structures currently implemented are
 
 .. module:: statsmodels.genmod.cov_struct
+   :synopsis: Covariance structures for GEE and GLM
```
Comment (josef-pkt) [inline docs/source/gee.rst]: only for GEE  (GLM assumes independent observation for parameter estimation)

File: docs/source/gee.rst [hunk 2]
```diff
@@ -103,6 +106,8 @@ Families
 The distribution families are the same as for GLM, currently implemented are
 
 .. module:: statsmodels.genmod.families.family
+   :synopsis: Generalized Linear Model (GLM) link families
```
Comment (josef-pkt) [inline docs/source/gee.rst]: drop word `link`   
families are the distributions, 
links are for the nonlinear specification of the mean function

File: docs/source/iolib.rst [hunk 3]
```diff
@@ -20,7 +20,7 @@ Module Reference
 ----------------
 
 .. module:: statsmodels.iolib
-
+   :synopsis: Tools for reading datasets
```
Comment (josef-pkt) [inline docs/source/iolib.rst]: and producing summary output

(I don't know how long the synopsis can be)

File: docs/source/multivariate.rst [hunk 4]
```diff
@@ -1,4 +1,6 @@
 .. module:: statsmodels.multivariate
+   :synopsis: Models for multiple series
```
Comment (josef-pkt) [inline docs/source/multivariate.rst]: "Models for multivariate data"
would be more precise

File: docs/source/stats.rst [hunk 5]
```diff
@@ -38,6 +42,8 @@ Residual Diagnostics and Specification Tests
    expected_robust_kurtosis
 
 .. module:: statsmodels.stats.diagnostic
+   :synopsis: Statistical methods and tests to diagnose model fit probelms
```
Comment (josef-pkt) [inline docs/source/stats.rst]: typo "problems"

File: docs/source/stats.rst [hunk 6]
```diff
@@ -75,6 +81,8 @@ Outliers and influence measures
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. module:: statsmodels.stats.outliers_influence
+   :synopsis: Statistical methods and tests measure outlier influence
```
Comment (josef-pkt) [inline docs/source/stats.rst]: "Statistical methods and measures for outliers and influence"

File: docs/source/stats.rst [hunk 7]
```diff
@@ -125,6 +133,8 @@ Goodness of Fit Tests and Measures
 some tests for goodness of fit for univariate distributions
 
 .. module:: statsmodels.stats.gof
+   :synopsis: Statistical methods and tests measure goodness of fit
```
Comment (josef-pkt) [inline docs/source/stats.rst]: Is alphabetical sorting relevant here?, e.g.
"Goodness of fit measures and tests"
versus
"Statistical methods for goodness of fit measures and tests"

File: docs/source/stats.rst [hunk 8]
```diff
@@ -221,6 +239,8 @@ to one way ANOVA, but still in developement
    pairwise_tukeyhsd
 
 .. module:: statsmodels.stats.multitest
+   :synopsis: False discovery rate adjustments
```
Comment (josef-pkt) [inline docs/source/stats.rst]: Multiple testing p-value adjustments

FDR is one group of those

File: docs/source/stats.rst [hunk 9]
```diff
@@ -398,6 +426,8 @@ These are utility functions to convert between central and non-central moments,
 kurtosis and cummulants.
 
 .. module:: statsmodels.stats.moment_helpers
+   :synopsis: Tools to compute moments from data
```
Comment (josef-pkt) [inline docs/source/stats.rst]: the data part is not handled

"Tools for converting moments"

File: docs/source/stats.rst (hunk 7 above)
Comment (bashtage) [inline docs/source/stats.rst]: No, just not lots of inspiration for writing ~100 synopses

**Comment by josef-pkt** [comment]
is this ready to merge?

(I never looked at what synopsis does or how module index build works)

**Comment by bashtage** [comment]
I just follow sphinx examples.  As far as I can see it is harmless.

**Comment by bashtage** [comment]
It is ready.

**Comment by josef-pkt** [comment]
In it goes

Thanks

**Pull request merged**